sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPSensor.java

```diff
@@ -107,10 +107,12 @@ public void execute(SensorContext context) {
       fileSystem.predicates().hasLanguage(Php.KEY));
 
     PHPAnalyzer phpAnalyzer = new PHPAnalyzer(ImmutableList.copyOf(checks.all()));
-    Iterable<InputFile> inputFiles = fileSystem.inputFiles(mainFilePredicate);
+
+    List<InputFile> inputFiles = new ArrayList<>();
+    fileSystem.inputFiles(mainFilePredicate).forEach(inputFiles::add);
 
     ProgressReport progressReport = new ProgressReport("Report about progress of PHP analyzer", TimeUnit.SECONDS.toMillis(10));
-    progressReport.start(Lists.newArrayList(fileSystem.files(mainFilePredicate)));
+    progressReport.start(inputFiles.stream().map(InputFile::filename).collect(Collectors.toList()));
```

Comment:
I think that `filename` only gives the name of the file. `toString` may return the full path, according to its javadoc:
```
Return a string to identify this file (suitable for logs).
```